src/inmanta/moduletool.py

```diff
@@ -1272,7 +1272,10 @@ def release(
         requested_version_bump: Optional[ChangeType] = ChangeType.parse_from_bools(revision, patch, minor, major)
         if not requested_version_bump and dev:
             # Dev always bumps
-            requested_version_bump = ChangeType.PATCH
+            if hasattr(module.metadata, "four_digit_version") and module.metadata.four_digit_version:
```

Comment:
Why do we need `hasattr` here? Same further down.

Reply:
without it I had issues with the v1 modules I think. let me test again

Reply:
Oh right, this is for either. I have a strong preference for an `isinstance` though.

Reply:
Or alternatively we could add the field to v1 metadata as an internal field, as `Literal[False]`. I'm not sure if that would be better or worse.

Reply:
i'll change it to isinstance